### PR TITLE
✅ Add example docker-compose file

### PR DESCRIPTION
**Pull request by deifyed**
+ Some minor improvements to the README. 

To test:
Follow the instructions in Usage -> docker-compose

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+version: '3.7'
```
Comment (yngvark) [inline docker-compose.yaml:1]: 3.8 exists I think. But doesn't matter really.

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  gatekeeper:
+    depends_on:
+      - redis
+    image: docker.pkg.github.com/oslokommune/gatekeeper/gatekeeper:1.0.25
+    networks:
+      - exampleapp
+    ports:
+      - '4554:4554'
+    environment:
+      BASE_URL: http://localhost:4554
```
Comment (yngvark) [inline docker-compose.yaml:19]: Consider adding
```
# See readme for documentation and examples
```

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      CLIENT_ID: ${GATEKEEPER_CLIENT_ID}
+      CLIENT_SECRET: ${GATEKEEPER_CLIENT_SECRET}
+      DISCOVERY_URL: ${GATEKEEPER_DISCOVERY_URL}
+      ORIGIN_WHITELIST: 'http://localhost:8080'
+      REDIS_URI: redis://redis:6379
+      TOKEN_COOKIES_SECURE: 'false' # Do not set this in production. This is for developing only.
+      UPSTREAMS: >
+        example=http://example-service:3000
+
+        #example-service:
```
Comment (yngvark) [inline docker-compose.yaml:29]: The indentation should be reduced somewhat.

File: docker-compose.yaml (hunk 1 above)
Comment (deifyed) [reply docker-compose.yaml:1]: Will update it the next time I work on the file. Cba testing it again just for this bump :p

**Pull request merged**